packages/utils/telemetry-utils/src/errorLogging.ts

```diff
@@ -394,7 +393,7 @@ export class LoggingError
 	extends Error
 	implements ILoggingError, Omit<IFluidErrorBase, "errorType">
 {
-	private _errorInstanceId = uuid();
+	private _errorInstanceId: string = crypto.randomUUID();
```

Comment:
TypeScript gives a type of `${string}-${string}-${string}-${string}-${string}` here by default now, which breaks, so I explicitly typed this as string to make the prior behavior.